### ISSUE TITLE
Code128 error when is number between characters

### ISSUE THREAD

**Issue by eLko99** [dementeddevil/BarcodeRenderingFramework#7]
Hello. I have an issue with generating Code128.
I have generated Code128 with value: 15a1s1a
but when i read that barcode, it says: 15a±s±a
I have ZenBarcode installed from visual studio NuGet

**Comment by dplx11** [comment]
That's true - i created round about 4.000 codes - every code with numbers between letters gets destroyed by special characters.
